src/Select/select/Item.ts

```diff
@@ -0,0 +1,17 @@
+export type OptionProps = Omit<
+  React.DetailedHTMLProps<
+    React.OptionHTMLAttributes<HTMLOptionElement>,
+    HTMLOptionElement
+  >,
+  "children"
+> & { children: string };
+
+export type OptgroupProps = React.DetailedHTMLProps<
+  React.OptgroupHTMLAttributes<HTMLOptGroupElement>,
+  HTMLOptGroupElement
+>;
+
+/**
+ * Represents the type of the item to be passed to `items` in `useSelect`
+ */
+export type Item = OptionProps;
```

Comment:
NBD It might be clearer to leave this as OptionProps? Item sounds like it's something more.
```suggestion
```

Reply:
This _felt_ more right as an abstraction before I was just using `OptionProps` which isn't even right, because the `children` is restricted. I'm axing this whole type file; thanks for making me re-think my assumptions!